README.md

```diff
@@ -289,6 +289,24 @@ A callback will be made on the AddressCallbackUrl when the user swipes to accept
 To ensure no unauthorized calls to your callback endpoints, we strongly suggest you apply either (or both) of these limitations:
 1. Firewall rules. These are the IP-ranges that MobilePay backend will be calling from: 212.93.32.0/19 and 185.218.228.0/22
 2. SSL "Common name" inspection. The "Common name" in our SSL client certificate should never change. Even when the certificate itself changes, or is issued to a different root.
+
+## Invalidation of a payment
+In some cases the user goes back to the merchant webshop and adds something to the shopping basket after the payment has been initiated. This could end up with several requests for the user with different amounts. This is why, the payment can be invalidated. There is an endpoint in the Online api to invalidate payments. Invalidation means that the user cannot create a request for the payment or accept the payment. Active requests will also expired immediately.
+
+The invalidation request will be processed in the MobilePay backend according to these rules:
+1. If a successful authorization already exists on the payment, the invalidation will return an error.
+2. If a callback has been sent, we will wait 30 seconds for a patch to the authorization attempt. A failed authorization will result in an invalidation, but a successful authorization will return an error.
+3. If we do not receive the patch within 30 seconds, we will accept the invalidation.
+4. When the invalidation is completed, the user cannot request or accept this payment.
+
+### Invalidation before callback
+If the invalidation request is sent before the callback has been received, the callback will not be sent and all that is needed is to call the invalidation endpoint.
+![Invalidation before callback](./assets/invalidation-before-callback.svg)
+
+### Invalidation after callback
+If the invalidation request is sent after the callback has been received, it is needed to call the invalidation endpoint as well as to patch the authorization attempt with error code 1010 - RejectedForInvalidatePayment. 
```

Comment:
to be more clear:
If the invalidation request is received by MobilePay after the callback to the PSP is performed, the PSP must also PATCH the authorizationAttempt with succeeded=false and the new ReasonCode 1010 - RejectedForInvalidatePayment.

Reply:
Fixed in commit 2. I removed "new" from your suggestion.